data/tribes/initialization/empire/units.lua

```diff
@@ -1300,34 +1394,62 @@ wl.Descriptions():new_tribe {
       {
          name = "empire_charcoal_kiln",
          helptexts = {
+            -- TRANSLATORS: Lore helptext for an empire production site: Charcoal Kiln
+            lore = pgettext("empire_building",
+               "‘Creta an carbone notandi?’<br>(Are they to be marked with chalk or charcoal?)"),
+            -- TRANSLATORS: Lore author helptext for an empire production site: Charcoal Kiln
+            lore_author = pgettext("empire_building", "I.e. judge whether they are good or bad"),
             -- TRANSLATORS: Purpose helptext for an empire production site: Charcoal Kiln
             purpose = pgettext("building", "Burns logs into charcoal.")
          }
       },
       {
          name = "empire_smelting_works",
          helptexts = {
+            -- TRANSLATORS: Lore helptext for an empire production site: Smelting Works
+            lore = pgettext("empire_building",
+               "‘Ignis aurum probat, miseria fortes viros.’<br>(Fire tests gold, adversity [tests] brave men.)"),
+            -- TRANSLATORS: Lore author helptext for an empire production site: Smelting Works
+            lore_author = pgettext("empire_building", "Saledus on the importance of experienced soldiers"),
```

Comment:
the lore and especially the author do not fit very well to the smelting works. 
may we find something like in front of the fire stone turns into wealth / power. 
author could be the ecobomic handbook stone tablet xx

Reply:
I thought it fits because gold is only used for advanced military equipment, so this draws a parallel with its production and use.

Reply:
or how about:

- Non omne, quod nitet, aurum est / not all that shines is gold
- nunc iacet aes, aurum in summum successit honorem / Now lies the brass, the gold has climbed to the highest honor
- Video ferrum ex isdem tenebris esse prolatum, quibus aurum / I see that iron was produced from the same darkness as gold

Reply:
the first one is a very common proverb even in german so it is easy to recognize and to translate, so it is definitly well suited for the smelting works.
However personally i really would love the third one, as it has more deepness and sounds a bit mysterious. In the end both would be ok.#, so your decision